src/config.c

```diff
@@ -990,12 +997,27 @@ void configGetCommand(client *c) {
     }
 
     di = dictGetIterator(matches);
-    addReplyMapLen(c, dictSize(matches));
+    unsigned long matchesSize = dictSize(matches);
+    addReplyMapLen(c, matchesSize);
+
+    struct {
+        const char *key;
+        sds value;
+    } *sorted = zmalloc(sizeof(*sorted) * matchesSize);
+    unsigned long sortedIndex = 0;
```

Comment:
For local variables, we normally use snake case like `sorted_index` or just sort names like `i`. Since the scope is pretty small here, I don't mind that we reuse the `i` variable already declared in this function.

Also matchesSize could be named `matches_size`, `num_matches` or just `n`.

If we use `i` and `n`, then the for loop below will be the typical short C style `for (i = 0; i < n; i++)`. I kind of like it. (Of course this brief style only works in simple functions without risk of confusion, so here it works well IMHO.)